src/test/java/com/knightcharacter/app/integration/SkillResourceIntegrationTest.java

```diff
@@ -0,0 +1,365 @@
+import static com.knightcharacter.app.TestConstants.buildJsonPathToIdInListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToLength;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToName;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToNameInBonusListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToNameInBonusListNestedInSkillListByIndexes;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToNameInListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToRarityInBonusListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToRarityInBonusListNestedInSkillListByIndexes;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToSkillBoostInBonusListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToSkillBoostInBonusListNestedInSkillListByIndexes;
+import static com.knightcharacter.app.TestConstants.buildPutSkillByIdUrl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.knightcharacter.app.factories.BonusFactory;
+import com.knightcharacter.app.factories.SkillFactory;
+import com.knightcharacter.app.gateway.privatedb.repository.BonusRepository;
+import com.knightcharacter.app.gateway.privatedb.repository.SkillRepository;
+import com.knightcharacter.app.gateway.privatedb.representation.Bonus;
+import com.knightcharacter.app.gateway.privatedb.representation.Skill;
+import com.knightcharacter.app.rest.request.SkillRequestDto;
+
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class SkillResourceIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private SkillRepository skillRepository;
+
+    @Autowired
+    private BonusRepository bonusRepository;
+
+    @AfterEach
+    void tearDown() {
```

Comment:
does this name explains the method functionality in the best way?

Reply:
Maybe not, but it is standard generated name of Intellij. BeforeEach = setUp, AfterEach = tearDown

Reply:
It's new information for me, thanks. I thinks that it's a good choise to use the standart one.

Reply:
Ok, I agree)